package.json

```diff
@@ -26,31 +26,31 @@
     },
     "license": "Apache-2.0",
     "devDependencies": {
-        "@aws-sdk/client-sqs": "3.391.0",
+        "@aws-sdk/client-sqs": "3.514.0",
```

Comment:
Tuhle verzi máme v lib-backend, tak bych ji tady zafixoval taky, ne? (Ale šťastnej z toho nejsem, ideální by bylo kdyby to stačilo udělat na jednom místě. 🤔)

Reply:
peerDependency 🤷‍♂️

Reply:
No v peer dependency to máme, ale tam to musí být právě taky na fixní verzi. Asi 🤔

Reply:
no jako u těch aws knihoven bych to ale nechal .. tam je to o držku

Reply:
No to jo, to je jasný. Já se jen snažím vymyslet jak zabránit tomu abysme na tý jedný verzi (která už teď je asi o 200 releasů zastaralá) zůstali další rok, protože když to budeme chtít updatovat v jedný servise, tak se kvůli tomu bude muset upravit dalších X repozitářů. 😕

Reply:
no .. tak vlastně nemusíme lockovat tu verzi, ale nebudem mít jistotu, že to na tý verzi funguje no .. a tak updatovat to v x repozitářích budeš vždycky

Reply:
Jo no. Kdyby to aspoň v aws verzovali nějak inteligentně aby bylo jasný co je feature a co bugfix. Se jim nechce udržovat minor verze. 😒

Reply:
no tak to neovlivníme no

Reply:
ok :D